pyproject.toml

```diff
@@ -249,6 +251,7 @@ addopts = [
   "-rA",
   "--import-mode=importlib",
   "--doctest-modules",
+  "--new-first",
```

Comment:
This run brand-new tests first?

Reply:
yes, this is just for local development, it sorts them by modification date

Reply:
on github there's no modification time